src/deploy/hosting/convertConfig.ts

```diff
@@ -185,8 +186,9 @@ export async function convertConfig(
           if (foundEndpoint) {
             vRewrite.functionRegion = foundEndpoint.region;
           } else {
-            throw new FirebaseError(
-              `Unable to find a valid endpoint for function ${vRewrite.function}`
+            logLabeledWarning(
+              `hosting[${config.site}]`,
+              `Unable to find a valid endpoint for function \`${vRewrite.function}\`, but still including it in the config`
```

Comment:
I'm going to hold off on saying so, I think. If only because I'm not 100% sure that it _should_ be an error. I can kind of see a reason to let this to continue working, but printing a warning for now should be fine. If we do decide to make this an error again in the future, it'll be included in a breaking change